doc/ni_by_example_2.md

```diff
@@ -125,7 +125,7 @@ Here's what you need to know:
   * `(lambda (u v) (u + v))` yields a closure. There's a lot of ink spilled on what a closure is and does. For the purpose of this tutorial, you can think of a closure as a a function that has not been supplied its arguments.
```

Comment:
Lispers are insensitive to the cringing of others, probably as a matter of principle.